notebooks/byod.py

```diff
@@ -1,17 +1,18 @@
-# publish to: "terra-notebook-utils-tests" "test-byod"
+#publish to: "TEMP-byod-edits" "test-byod-local"
 import os
 import callysto
 
 # Mock the notebook environment
-os.environ['WORKSPACE_NAME'] = "terra-notebook-utils-tests"
-os.environ['WORKSPACE_BUCKET'] = "gs://fc-9169fcd1-92ce-4d60-9d2d-d19fd326ff10"
-os.environ['GOOGLE_PROJECT'] = "firecloud-cgl"
+os.environ['WORKSPACE_NAME'] = "TEMP-byod-edits"
```

Comment:
Why are these being changed?

Reply:
1. To push to my own workspace rather than clutter up Brians
2. To avoid Brian and I pushing notebooks on top of each other and not being able to keep track of whose is whose
3. To debug my persistent credential issues in a fresh workspace where I have owner access
4. To debug ' ' ' with a billing project that I'm allowed use directly, unlike firecloud-cgl